resources/assets/react/screens/Profile/AssociativeCareer.js

```diff
@@ -91,31 +77,33 @@ class AssociativeCareerScreen extends React.Component {
 			<div className="ml-5">
 				{rolesFetched &&
 					semestersFetched &&
-					Object.keys(associativeSemesters).map(semester_id => {
-						const semester = semesters.find(semester => semester.id === semester_id);
-
-						const assosBySemesterList = associativeSemesters[semester_id].slice().map(asso => {
-							const roleName = roles.find(role => role.id === asso.pivot.role_id).name;
-
-							return (
-								<NavLink key={asso.id} to={`/assos/${asso.login}`}>
-									<AssoCard
-										key={asso.id}
-										name={asso.name}
-										shortname={`${asso.shortname} - ${roleName}`}
-										image={asso.image}
-										login={asso.parent ? asso.parent.login : asso.login}
-									/>
-								</NavLink>
-							);
-						});
-
-						const title = assosBySemesterList ? (
-							<h2 style={{ margin: 20 }}>Semestre {semester.name}</h2>
-						) : null;
-
-						return assosBySemesterList ? [title, ...assosBySemesterList] : null;
-					})}
+					Object.keys(associativeSemesters)
+						.reverse()
+						.map(semester_id => {
+							const semester = semesters.find(semester => semester.id === semester_id);
+
+							const assosBySemesterList = associativeSemesters[semester_id].map(asso => {
+								const roleName = roles.find(role => role.id === asso.pivot.role_id).name;
+
+								return (
+									<NavLink key={asso.id} to={`/assos/${asso.login}`}>
+										<AssoCard
```

Comment:
You should use the MemberCard instead, so you can put a `title` (the shortname association) and a `description` (the role)